app/models/sequencing_request.rb

```diff
@@ -16,6 +16,7 @@ class SequencingRequest < CustomerRequest # rubocop:todo Style/Documentation
     custom_attribute(:fragment_size_required_to, integer: true, minimum: 1)
 
     custom_attribute(:read_length, integer: true, validator: true, required: true, selection: true)
+    custom_attribute(:requested_flowcell_type, string: true, required: true)
```

Comment:
This is only required for the novaseq request types, so required can be false. As long as the appropriate validator gets added to the novaseq request types then I *think* that will handle that for you. (As '' won't be in the list)